packages/core/src/clients/AbstractClient.ts

```diff
@@ -263,16 +258,6 @@ export class AbstractQueryClient implements IAbstractQueryClient {
     )
   }
 
-  /**
-   * Query the Abstract API.
-   * @param params
-   */
-  public async queryApi<TResult, V extends Variables = Variables>(
```

Comment:
Why remove this method? Found it handy as a dev

Reply:
that file is planned to be deleted, all the legacy support will be in another directory.